BonMot.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "BonMot"
-  s.version          = "2.4.1"
+  s.version          = "3.0"
```

Comment:
#req please don't bump the version. This _will_ go into 3.0, but I prefer to save the version bump for when I actually tag a release.